package.json

```diff
@@ -18,6 +18,7 @@
     "publish-storybook": "bash .scripts/publish_storybook.sh"
   },
   "devDependencies": {
+    "@kadira/storybook": "^1.35.1",
```

Comment:
it's just not there
https://github.com/kadirahq/react-storybook-addon-info/blob/master/package.json